bin/release.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+function usage {
+    echo
+    echo "Builds all the images for the specified kimai tag and then tags them and pushes"
+    echo "them:"
+    echo "  fpm-alpine-1.10.2-prod, latest, latest-prod"
```

Comment:
replace `1.10.2` with `X.Y.z` if you want to remove all version numbers?

Reply:
Good idea